translation/en/RedirectedResources/assets/abdata/adv/scenario/c36/penetration/masturbation_found/translation.txt

```diff
@@ -1,4 +1,4 @@
 //
 // Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
 //
-「ふふっ、オナニー中に襲ってこないという事は、躾が行き渡ってるという事かしら？」=“Fufu, the fact that you didn't assault me during masturbation means that you've been trained well, right?”
+「ふふっ、オナニー中に襲ってこないという事は、躾が行き渡ってるという事かしら？」=“Fufu--, The fact that you didn't assault me during masturbation, is it because the training's taken root well, I wonder.”
```

Comment:
```suggestion
「ふふっ、オナニー中に襲ってこないという事は、躾が行き渡ってるという事かしら？」=“Fufu-- The fact that you didn't assault me during masturbation, is it because the training's taken root well, I wonder.”
```